QUALITY_DECLARATION.md

```diff
@@ -120,7 +120,12 @@ A description of how coverage statistics are summarized from this page, can be f
 ### Performance [4.iv]
 
 `rcutils` follows the recommendations for performance testing of C code in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#performance), and opts to do performance analysis on each release rather than each change.
-It is not yet defined if this package requires performance testing and how addresses this topic.
+
+System level performance benchmarks that cover features of `rcutils` can be found at:
+* [Benchmarks](http://build.ros2.org/view/Rci/job/Rci__benchmark_ubuntu_focal_amd64/lastCompletedBuild/)
```

Comment:
How about linking to the benchmark tables instead? The benchmark results are more useful to see build-over-build than just the latest build.

http://build.ros2.org/view/Rci/job/Rci__benchmark_ubuntu_focal_amd64/BenchmarkTable/

Reply:
Ooh, good call!